dChatServer/ChatPacketHandler.cpp

```diff
@@ -132,8 +125,8 @@ void ChatPacketHandler::HandleFriendRequest(Packet* packet) {
 	// If at this point we dont have a target, then they arent online and we cant send the request.
 	// Send the response code that corresponds to what the error is.
 	if (!requestee) {
-		requestee.playerName = playerName;
-		auto responseType = Database::Get()->GetCharacterInfo(playerName)
+		requestee.playerName = playerName.GetAsString();
```

Comment:
There is a lot of copying of the LUWString.GetAsString going on here.  Consider a `const auto nameAsStr = playerName.GetAsString();`

Reply:
yeah, true